documentation/README.md

```diff
@@ -2,6 +2,8 @@
 This is the Rogue API, it is used to capture activity from members.
 All `POST` and `PUT` endpoints require an api key (`X-DS-Rogue-API-Key`) in the header to be submitted with the request. 
 
+Refer to this [document](https://github.com/DoSomething/rogue/wiki/API) for the most updated documentation. 
```

Comment:
ah gotcha @sbsmith86 ! Just added more robust documentation and linked the documentation from the wiki.